client/src/ui/organisms/footer/index.scss

```
@@ -43,6 +44,10 @@
       margin-right: 10px;
     }
 
+    ul {
+      padding-inline-start: 5px;
```

We work in increments of `6px` to keep spacing consistent and use SASS variables and a little math for this so here, please do:

```suggestion
      padding-inline-start: $base-spacing / 4;
```